go.mod

```diff
@@ -140,7 +141,6 @@ require (
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20191115221424-83cc0476cb11
 	google.golang.org/grpc v1.25.1 // indirect
-	gopkg.in/goracle.v2 v2.16.3 // indirect
```

Comment:
why has this been removed?

Reply:
It has been removed from master in the meantime: https://github.com/elastic/beats/pull/15683